src/main/java/io/vertx/rabbitmq/RabbitMQClient.java

```diff
@@ -42,6 +42,18 @@ static RabbitMQClient create(Vertx vertx, JsonObject config) {
   //TODO: Think about splitting this out into different API's with specific roles (admin, pub, sub)
   //TODO: Simplify/Change name of API methods to match more vert.x type verbiage ?
 
+  /**
+   * Limit the amount of unacknowledged messages on a connection.
+   * <p>
+   * More: https://www.rabbitmq.com/consumer-prefetch.html
+   *
+   * @param prefetchCount amount of unacknowledged messages
+   * @param global        true if the settings should be applied to the entire channel rather than each consumer
+   * @param resultHandler operation status handler
+   * @see com.rabbitmq.client.Channel#basicQos(int)
+   */
+  void basicQos(int prefetchCount, boolean global, Handler<AsyncResult<Void>> resultHandler);
```

Comment:
shouldn't it be fluent ?

Reply:
@vietj,  I think it should, but I'm wondering why all of the `RabbitMQClient` methods are not fluent.

So maybe it worth it to make the whole Client fluent in a related PR?

WDYT?

Reply:
yes for the operations that make sense